index.html

```diff
@@ -922,18 +926,25 @@ <h3>Processing Model</h3>
 <!-- Source: https://docs.google.com/document/d/1I7XGNJ57Qgjkg9pL11s7MK7zGEcwAgdNj1W5f7NKbW8/ -->
 <img src="timestamp-diagram.svg" alt="Resource Timing attributes"
 style='margin-top: 1em'></figure>
-<p>For each resource <a data-cite="FETCH#concept-fetch">fetched</a>
-by the current <a data-cite="HTML#browsing-context">browsing
-context</a>, excluding resources fetched by cross-origin
-stylesheets fetched with <code>no-cors</code> policy, perform the
-following steps:</p>
+
+<p>For each resource which <a>Request</a> has a non-null <a>client</a>, perform
+the following steps:</p>
+<ol data-link-for="PerformanceResourceTiming">
+<li>If the resource is <a data-cite="Fetch#concept-fetch">fetched</a> by a
+cross-origin stylesheet which was fetched with <code>no-cors</code> policy, 
+abort the remaining steps.</li>
 <p class="issue">Above cross-origin exclusion should be defined via
 Fetch registry: CSS needs to be defined in terms of Fetch and set
 some kind of "opaque request flag" for no-CORS CSS subresources. In
 turn, Resource Timing should interface with Fetch registry to
 surface resource fetch events.</p>
 <div class="issue" data-number="27"></div>
-<ol data-link-for="PerformanceResourceTiming">
+<li>If the resource's <a>Request</a>'s
+<a data-cite="Fetch#concept-request-destination">destination</a> equals to
+"document", and was not triggered by 
```

Comment:
"and Request was not triggered", right?  The current wording is talking  about the destination being triggered...

Reply:
yup